testfixtures/rmtree.py

```diff
@@ -28,12 +33,13 @@ def _waitfor(func, pathname, waitall=False):
             # Other Windows APIs can fail or give incorrect results when
             # dealing with files that are pending deletion.
             L = os.listdir(dirname)
-            if not (L if waitall else name in L): # pragma: no branch
+            if not (L if waitall else name in L):  # pragma: no branch
                 return
             # Increase the timeout and try again
-            time.sleep(timeout) # pragma: no cover
-            timeout *= 2 # pragma: no cover
-        warnings.warn('tests may fail, delete still pending for ' + pathname, # pragma: no cover
+            time.sleep(timeout)  # pragma: no cover
```

Comment:
I really don't like the double space before comments in the same line. Yes, it's in the pep, but I think it's ugly.